src/physics/PhysicsList.cc

```diff
@@ -33,6 +40,55 @@ void PhysicsList::ConstructProcess() {
   AddParameterization();
   Shielding::ConstructProcess();
   ConstructOpticalProcesses();
+  EnableThermalNeutronScattering();
+}
+
+// Add thermal neutron scattering to this physics list.
+// Based on the technique shown here: http://tinyurl.com/hszjtln
```

Comment:
Would prefer the full URLs, for clarity/posterity. (And actually, would rather keep all method-level documentation in the header for consistency/Doxygen.)